src/olympia/devhub/tests/test_views.py

```diff
@@ -3835,3 +3835,29 @@ def test_devhub_version_edit_page(self):
     def test_devhub_version_list_page(self):
         url = reverse('devhub.addons.versions', args=[self.addon.slug])
         self.assertNameAndNoXSS(url)
+
+
+class TestSwitchToUnlisted(UploadTest):
```

Comment:
Tests for the versions and status page, I guess.  Anyway, I'll leave it to your judgement where is best to add, but tests such as `TestVersion.test_user_can_unlist_addon` already test the procedure.